packages/@aws-cdk/aws-docdb/lib/cluster.ts

```diff
@@ -257,20 +257,70 @@ export class DatabaseCluster extends DatabaseClusterBase {
    */
   public static fromDatabaseClusterAttributes(scope: Construct, id: string, attrs: DatabaseClusterAttributes): IDatabaseCluster {
     class Import extends DatabaseClusterBase implements IDatabaseCluster {
-      public readonly defaultPort = ec2.Port.tcp(attrs.port);
+      public readonly clusterIdentifier = attrs.clusterIdentifier;
+      public readonly defaultPort = attrs.port ? ec2.Port.tcp(attrs.port) : undefined;
       public readonly connections = new ec2.Connections({
-        securityGroups: [attrs.securityGroup],
+        securityGroups: !!attrs.securityGroup ? [attrs.securityGroup] : undefined,
```

Comment:
Why the double `!!` here, all of a sudden? Removing it will not change this code.